iree/testing/BUILD

```diff
@@ -6,12 +6,18 @@
 
 # Testing utilities for IREE.
 
+load("//iree:build_defs.oss.bzl", "iree_cmake_extra_content")
+
 package(
     default_visibility = ["//visibility:public"],
     features = ["layering_check"],
     licenses = ["notice"],  # Apache 2.0
 )
 
+iree_cmake_extra_content(
+    content = "set(IREE_DISABLE_INSTALL_HEADERS ON)",
```

Comment:
The `set(IREE_DISABLE_INSTALL_HEADERS ON)` line needs to be added by hand to the corresponding `CMakeLists.txt` which doesn't use bazel_to_cmake:

https://github.com/google/iree/blob/f2ce179dd0a539858f035bd31e4b5ac77108991a/iree/testing/CMakeLists.txt#L7

Right now, `iree/testing/benchmark.h` gets installed.